src/components/ViewportButton/index.tsx

```diff
@@ -0,0 +1,80 @@
+import React, { ReactNode } from "react";
+import { Button, ButtonProps, Tooltip } from "antd";
+import classNames from "classnames";
+
+import { TOOLTIP_COLOR } from "../../constants";
+import styles from "./style.css";
+
+interface ViewportButtonProps extends ButtonProps {
+    tooltipText?: string;
+    tooltipPlacement?: "top" | "bottom" | "left" | "right";
```

Comment:
is it possible to also extend the tooltip props instead of re-writing it here?

Reply:
yes it is! didn't realize how till you suggested it, thanks.